src/components/ui/FlowWindow.tsx

```
@@ -0,0 +1,440 @@
+        title: 'Flow Tutorial',
+        type: 'tutorial'
+    }
+];
+
+const FlowWindow: React.FC<FlowWindowProps> = ({ isVisible, onClose }) => {
+    const [activeSection, setActiveSection] = useState('last10');
+    
+    // Get current values from the store including trend history and flow points
+    const { 
+        resourceTrendHistory, 
+        resourceTrends, 
```

ESLint

[error] 'resourceTrends' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)